ongoing/scrabble-score/scrabble_score.go

```diff
@@ -1 +1,35 @@
 package scrabble
+
+import "strings"
+
+// GetScore function using case to set value for each letter
+func GetScore(r rune) int {
+	switch r {
+	case 'A', 'E', 'I', 'L', 'N', 'O', 'R', 'S', 'T', 'U':
+		return 1
+	case 'D', 'G':
+		return 2
+	case 'B', 'M', 'C', 'P':
+		return 3
+	case 'F', 'H', 'V', 'W', 'Y':
+		return 4
+	case 'K':
+		return 5
+	case 'J', 'X':
+		return 8
+	case 'Q', 'Z':
+		return 10
+	default:
+		return 0
+	}
+}
+
+// Score takes a word and returns the value of that word
+// according to scrabble scoring rules
+func Score(w string) int {
+	score := 0
```

Comment:
Minor comment, nothing wrong with this at all, but you can equally do `var score int` for the exact same effect